src/lang/en.json

```diff
@@ -28,7 +28,7 @@
       },
       "right-margin": {
         "Name": "Drawer Only: Right Margin",
-        "Hint": "Sets the offest from the sidebar to the right side of the GM Screen. This number affects the maximum possible width of the screen in drawer display mode."
+        "Hint": "The offest from the sidebar to the right side of the GM Screen. This number affects the maximum possible width of the screen in drawer display mode."
```

Comment:
The problem here is that `offset` is misspelled, not that the line begins with `The`

Reply:
Ah damn, I didn't catch that, I'll fix it